spark-icons/dependencies/releaseRuntimeClasspath.txt

```diff
@@ -0,0 +1,49 @@
+androidx.activity:activity-ktx:1.5.1
```

Comment:
All this with only 2 dependencies 🥵

Reply:
It contains more than the 2 dependencies of the build file, because some of them comes from the `android-compose` plugin:



<details><summary>👁️ `androidx.compose.ui:ui`…</summary>

```
releaseRuntimeClasspath - Runtime classpath of compilation 'release' (target  (androidJvm)).
+--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.10
|    +--- org.jetbrains.kotlin:kotlin-stdlib:1.8.10
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.8.10
|    |    \--- org.jetbrains:annotations:13.0
|    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.8.10
|         \--- org.jetbrains.kotlin:kotlin-stdlib:1.8.10 (*)
+--- androidx.compose:compose-bom:2023.01.00
|    +--- androidx.compose.ui:ui:1.3.3 (c)
|    +--- androidx.compose.runtime:runtime:1.3.3 (c)
|    +--- androidx.compose.runtime:runtime-saveable:1.3.3 (c)
|    +--- androidx.compose.ui:ui-geometry:1.3.3 (c)
|    +--- androidx.compose.ui:ui-graphics:1.3.3 (c)
|    +--- androidx.compose.ui:ui-text:1.3.3 (c)
|    +--- androidx.compose.ui:ui-unit:1.3.3 (c)
|    \--- androidx.compose.ui:ui-util:1.3.3 (c)
+--- androidx.compose.ui:ui -> 1.3.3
|    +--- androidx.activity:activity-ktx:1.5.1
|    |    +--- androidx.activity:activity:1.5.1
|    |    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0
|    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.8.10 (*)
|    |    |    +--- androidx.collection:collection:1.0.0 -> 1.1.0
|    |    |    |    \--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    +--- androidx.core:core:1.8.0
|    |    |    |    +--- androidx.annotation:annotation:1.2.0 -> 1.5.0 (*)
|    |    |    |    +--- androidx.annotation:annotation-experimental:1.1.0
|    |    |    |    +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    |    |    +--- androidx.concurrent:concurrent-futures:1.0.0
|    |    |    |    |    +--- com.google.guava:listenablefuture:1.0
|    |    |    |    |    \--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    |    +--- androidx.lifecycle:lifecycle-runtime:2.3.1 -> 2.5.1
|    |    |    |    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    |    |    +--- androidx.arch.core:core-common:2.1.0
|    |    |    |    |    |    \--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    |    |    +--- androidx.arch.core:core-runtime:2.1.0
|    |    |    |    |    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    |    |    |    \--- androidx.arch.core:core-common:2.1.0 (*)
|    |    |    |    |    \--- androidx.lifecycle:lifecycle-common:2.5.1
|    |    |    |    |         +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    |    |         \--- androidx.lifecycle:lifecycle-common-java8:2.5.1 (c)
|    |    |    |    \--- androidx.versionedparcelable:versionedparcelable:1.1.1
|    |    |    |         +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    |         \--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    |    +--- androidx.lifecycle:lifecycle-runtime:2.5.1 (*)
|    |    |    +--- androidx.lifecycle:lifecycle-viewmodel:2.5.1
|    |    |    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.6.21 -> 1.8.10 (*)
|    |    |    |    \--- androidx.lifecycle:lifecycle-viewmodel-savedstate:2.5.1 (c)
|    |    |    +--- androidx.lifecycle:lifecycle-viewmodel-savedstate:2.5.1
|    |    |    |    +--- androidx.annotation:annotation:1.0.0 -> 1.5.0 (*)
|    |    |    |    +--- androidx.core:core-ktx:1.2.0 -> 1.5.0
|    |    |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.31 -> 1.8.10 (*)
|    |    |    |    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    |    |    \--- androidx.core:core:1.5.0 -> 1.8.0 (*)
|    |    |    |    +--- androidx.lifecycle:lifecycle-livedata-core:2.5.1
|    |    |    |    |    +--- androidx.arch.core:core-common:2.1.0 (*)
|    |    |    |    |    +--- androidx.arch.core:core-runtime:2.1.0 (*)
|    |    |    |    |    \--- androidx.lifecycle:lifecycle-common:2.5.1 (*)
|    |    |    |    +--- androidx.lifecycle:lifecycle-viewmodel:2.5.1 (*)
|    |    |    |    +--- androidx.savedstate:savedstate:1.2.0
|    |    |    |    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    |    |    +--- androidx.arch.core:core-common:2.1.0 (*)
|    |    |    |    |    +--- androidx.lifecycle:lifecycle-common:2.4.0 -> 2.5.1 (*)
|    |    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.6.20 -> 1.8.10 (*)
|    |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.6.21 -> 1.8.10 (*)
|    |    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.1 -> 1.6.4
|    |    |    |         +--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4
|    |    |    |         |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.6.4
|    |    |    |         |         +--- org.jetbrains.kotlinx:kotlinx-coroutines-bom:1.6.4
|    |    |    |         |         |    +--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4 (c)
|    |    |    |         |         |    +--- org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.6.4 (c)
|    |    |    |         |         |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4 (c)
|    |    |    |         |         +--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.21 -> 1.8.10 (*)
|    |    |    |         |         \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.6.21 -> 1.8.10
|    |    |    |         +--- org.jetbrains.kotlinx:kotlinx-coroutines-bom:1.6.4 (*)
|    |    |    |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.21 -> 1.8.10 (*)
|    |    |    +--- androidx.savedstate:savedstate:1.2.0 (*)
|    |    |    +--- androidx.tracing:tracing:1.0.0
|    |    |    |    \--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.6.21 -> 1.8.10 (*)
|    |    +--- androidx.core:core-ktx:1.1.0 -> 1.5.0 (*)
|    |    +--- androidx.lifecycle:lifecycle-runtime-ktx:2.5.1
|    |    |    +--- androidx.annotation:annotation:1.0.0 -> 1.5.0 (*)
|    |    |    +--- androidx.lifecycle:lifecycle-runtime:2.5.1 (*)
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.6.21 -> 1.8.10 (*)
|    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.1 -> 1.6.4 (*)
|    |    +--- androidx.lifecycle:lifecycle-viewmodel-ktx:2.5.1
|    |    |    +--- androidx.lifecycle:lifecycle-viewmodel:2.5.1 (*)
|    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.6.21 -> 1.8.10 (*)
|    |    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.1 -> 1.6.4 (*)
|    |    +--- androidx.savedstate:savedstate-ktx:1.2.0
|    |    |    +--- androidx.savedstate:savedstate:1.2.0 (*)
|    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.6.20 -> 1.8.10 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.6.21 -> 1.8.10 (*)
|    +--- androidx.annotation:annotation:1.5.0 (*)
|    +--- androidx.autofill:autofill:1.0.0
|    |    \--- androidx.core:core:1.1.0 -> 1.8.0 (*)
|    +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    +--- androidx.compose.runtime:runtime:1.3.3
|    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.8.10 (*)
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4 (*)
|    +--- androidx.compose.runtime:runtime-saveable:1.3.3
|    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    +--- androidx.compose.runtime:runtime:1.3.3 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.8.10 (*)
|    +--- androidx.compose.ui:ui-geometry:1.3.3
|    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    +--- androidx.compose.runtime:runtime:1.1.1 -> 1.3.3 (*)
|    |    +--- androidx.compose.ui:ui-util:1.3.3
|    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.8.10 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.8.10 (*)
|    +--- androidx.compose.ui:ui-graphics:1.3.3
|    |    +--- androidx.annotation:annotation:1.2.0 -> 1.5.0 (*)
|    |    +--- androidx.compose.runtime:runtime:1.1.1 -> 1.3.3 (*)
|    |    +--- androidx.compose.ui:ui-unit:1.3.3
|    |    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    |    +--- androidx.compose.runtime:runtime:1.1.1 -> 1.3.3 (*)
|    |    |    +--- androidx.compose.ui:ui-geometry:1.3.3 (*)
|    |    |    +--- androidx.compose.ui:ui-util:1.3.3 (*)
|    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.8.10 (*)
|    |    +--- androidx.compose.ui:ui-util:1.3.3 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.10 -> 1.8.10
|    +--- androidx.compose.ui:ui-text:1.3.3
|    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
|    |    +--- androidx.compose.runtime:runtime:1.2.0 -> 1.3.3 (*)
|    |    +--- androidx.compose.runtime:runtime-saveable:1.2.0 -> 1.3.3 (*)
|    |    +--- androidx.compose.ui:ui-graphics:1.3.3 (*)
|    |    +--- androidx.compose.ui:ui-unit:1.3.3 (*)
|    |    +--- androidx.compose.ui:ui-util:1.3.3 (*)
|    |    +--- androidx.core:core:1.7.0 -> 1.8.0 (*)
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.8.10 (*)
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.10 -> 1.8.10
|    |    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4 (*)
|    +--- androidx.compose.ui:ui-unit:1.3.3 (*)
|    +--- androidx.compose.ui:ui-util:1.3.3 (*)
|    +--- androidx.core:core:1.5.0 -> 1.8.0 (*)
|    +--- androidx.customview:customview-poolingcontainer:1.0.0
|    |    +--- androidx.core:core-ktx:1.5.0 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.6.21 -> 1.8.10 (*)
|    +--- androidx.lifecycle:lifecycle-common-java8:2.3.0 -> 2.5.1
|    |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |    \--- androidx.lifecycle:lifecycle-common:2.5.1 (*)
|    +--- androidx.lifecycle:lifecycle-runtime:2.3.0 -> 2.5.1 (*)
|    +--- androidx.lifecycle:lifecycle-viewmodel:2.3.0 -> 2.5.1 (*)
|    +--- androidx.profileinstaller:profileinstaller:1.2.0
|    |    +--- androidx.annotation:annotation:1.2.0 -> 1.5.0 (*)
|    |    \--- androidx.startup:startup-runtime:1.1.1
|    |         +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
|    |         \--- androidx.tracing:tracing:1.0.0 (*)
|    +--- androidx.savedstate:savedstate-ktx:1.2.0 (*)
|    +--- org.jetbrains.kotlin:kotlin-stdlib:1.7.10 -> 1.8.10 (*)
|    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.7.10 -> 1.8.10
|    +--- org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.4 (*)
|    \--- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4 (*)
\--- androidx.appcompat:appcompat-resources:1.5.1
     +--- androidx.annotation:annotation:1.2.0 -> 1.5.0 (*)
     +--- androidx.collection:collection:1.0.0 -> 1.1.0 (*)
     +--- androidx.core:core:1.6.0 -> 1.8.0 (*)
     +--- androidx.vectordrawable:vectordrawable:1.1.0
     |    +--- androidx.annotation:annotation:1.1.0 -> 1.5.0 (*)
     |    +--- androidx.core:core:1.1.0 -> 1.8.0 (*)
     |    \--- androidx.collection:collection:1.1.0 (*)
     \--- androidx.vectordrawable:vectordrawable-animated:1.1.0
          +--- androidx.vectordrawable:vectordrawable:1.1.0 (*)
          +--- androidx.interpolator:interpolator:1.0.0
          |    \--- androidx.annotation:annotation:1.0.0 -> 1.5.0 (*)
          \--- androidx.collection:collection:1.1.0 (*)
```

</details>

Reply:
I'll double check to see if we can use a more fine tuned dependency than this huge one 😭